<launch>
  <node name="wm_vision_kernel" pkg="wm_vision_kernel" type="wm_vision_kernel" launch-prefix="xterm -e gdb -ex  --args"/>

  <param name="debug_mode" type="bool" value="true" />
  <param name="cfvf_path" type="string" value="$(find wm_vision_kernel)/config/cfvf/" />
  <param name="catalog_path" type="string" value="$(find wm_vision_kernel)/config/catalog.info" />
</launch>
